src/unfuse.jl

```diff
@@ -24,13 +24,18 @@
 # 1. broadcast(F, x::Union{previous_types,T}...) = F(Broadcasted.(x)...).value
 # 2. bf(x::T) needs to be implemented (after being imported)
 
-type Broadcasted{T}
+mutable struct Broadcasted{T}
     value::T
 end
 getval(x::Broadcasted)=x.value
 # We need this to not override regular broadcast(f, A, Bs...):
-using Base.Broadcast: broadcast_c, containertype
-broadcast(f, x::Union{Number,AbstractArray}...)=broadcast_c(f, containertype(x...), x...)
+if VERSION < v"0.7.0-DEV.2635"
+    using Base.Broadcast: broadcast_c, containertype
+    broadcast(f, x::Union{Number,AbstractArray}...)=broadcast_c(f, containertype(x...), x...)
+else
+    using Base.Broadcast: combine_styles
```

Comment:
@denizyuret These are the changes related to broadcast. The change does what the deprecation warning suggests to do.

Reply:
But these are only used on 0.7 - so it is not clear to me which changes I made affect Autograd/Knet on 0.6 regarding broadcasting.